packages/dx-react-scheduler/src/types/grouping/grouping-state.types.ts

```diff
@@ -0,0 +1,22 @@
+import { Grouping, GroupKey, GroupOrientation } from '../index';
+
+// tslint:disable-next-line:no-namespace
+export interface GroupingStateProps {
+  /** Specifies resources to group by. */
+  grouping: Array<Grouping>;
```

Comment:
Why is grouping required property? According to the documentation, if I don't use grouping, grouping state would use `main resource`